dcicutils/ff_utils.py

```diff
@@ -457,65 +457,167 @@ def faceted_search(key=None, ff_env=None, item_type=None, **kwargs):
     return search_metadata(search, ff_env=ff_env, key=key)
 
 
-def get_associated_qc_metrics(uuid, key=None, ff_env=None,
-                              exclude_raw_files=True,
-                              exclude_supplementary_files=True):
+def fetch_files_qc_metrics(data, associated_files=['processed_files'],
+                           ignore_typical_fields=True,
+                           key=None, ff_env=None):
     """
-    Given a uuid of an experiment set, return a dictionary of uuid : item
-    mappings of quality metric items.
+    Utility function to grap all the qc metrics from associated types of file such as:
+    'proccessed_files', 'other_processed_files', 'files'
+    inputs:
+        data: the metadata of a ExperimentSet or Experiment
+        associated_files: a list of the types of the files fields the qc metrics will be extracted from:
+            examples are = ['files', 'processed_files', 'other_processed_files']
+    Args:
```

Comment:
I'd replace 'inputs' with 'Args' and delete the second heading. I get in principle what you're trying to do but I don't think its necessary